### DISCUSSION TITLE
If running multiple sidekiq services are they identifiable on the lifecycle events?

### DISCUSSION THREAD

**Discussion by cavi21** [sidekiq/sidekiq#6114] [Q&A]
I have a server that runs 2 sidekiq services `sidekiq-1` and `sidekiq-2` and have one config only (on a rails application), I want to plug something (start something on a thread and queue a job to be run as soon as the jobs are dispatched) on the `sidekiq-2` `:startup` event, is it possible? or how can I approach this?

Thanks!

**Comment by mperham** [comment]
That's really a question for systemd. AFAIK systemd creates identical copies of the same service. You can create a separate unit file or somehow customize the ENV for one of them and add the :startup block based on that ENV.

**Reply by cavi21** [reply]
hey @mperham thanks for getting back and what about using the `labels` config? is that information available when I run `Sidekiq.configure_server` on my initializer?

**Reply by cavi21** [reply]
I want to confirm that using the `labels` on the `yml` config file I was able to identify the process that needs to handle the custom startup event.
Thanks!
